tests/templates/requestTestData/trailingSlash.ts

```diff
@@ -0,0 +1,146 @@
+import type { TestSet } from '../../_helpers';
+import { mockPrerenderConfigFile } from '../../_helpers';
+import { createValidFuncDir } from '../../_helpers';
+
+// `trailingSlash` option in `next.config.js` is `true`.
+
+const rawVercelConfig: VercelConfig = {
+	version: 3,
+	routes: [
+		{
+			src: '^(?:/((?!\\.well-known(?:/.*)?)(?:[^/]+/)*[^/]+\\.\\w+))/$',
+			headers: { Location: '/$1' },
+			status: 308,
+			missing: [{ type: 'header', key: 'x-nextjs-data' }],
+			continue: true,
+		},
+		{
+			src: '^(?:/((?!\\.well-known(?:/.*)?)(?:[^/]+/)*[^/\\.]+))$',
+			headers: { Location: '/$1/' },
+			status: 308,
+			continue: true,
+		},
+		{
+			src: '/404/?',
+			status: 404,
+			continue: true,
+			missing: [{ type: 'header', key: 'x-prerender-revalidate' }],
+		},
+		{ src: '/500', status: 500, continue: true },
+		{ handle: 'resource' },
+		{ src: '/.*', status: 404 },
+		{ handle: 'rewrite' },
+		{ src: '^/(?<nxtPlang>[^/]+?)(?:/)?$', dest: '/[lang]?nxtPlang=$nxtPlang' },
```

Comment:
does `nxtPlang` has any meaning? could you maybe use something like generic or just `x` instead?

Reply:
That was just what was in the route config i copied it from. can change it to be more descriptive sure

Reply:
yeah no I just meant that that feels a bit random, anything like that makes it clear that it's a dummy/meaningless string would be better I think 🙂 

(I've seen a couple of this in your other tests, I guess this just stuck a bit with me because I don't what it could indicate 🤣 )